Makefile

```diff
@@ -33,7 +33,10 @@ usage:
 	@echo "  update_plutus       -- Update plutus version with niv"
 
 hoogle: requires_nix_shell
-	hoogle server --local
+	pkill hoogle || true
```

Comment:
Bit of a footgun, In case there is another Hoogle instance on another port hosting something else.

Reply:
True but I find the situation wherein one has multiple Hoogle projects running less frequent than the number of types one enters `make hoogle` and then gets shouted at. I'd be happy for @Geometer1729 to serve as final arbiter.

Reply:
I wasn't vetoing - there may be a more canonical way ,along the lines of `lsof :8080`.

Reply:
Oh, cool. My shell scripting is _weak_, so would welcome any suggestions.